currency/pair_methods.go

```diff
@@ -131,3 +131,17 @@ func (p Pair) Other(c Code) (Code, error) {
 	}
 	return EMPTYCODE, ErrCurrencyCodeEmpty
 }
+
+// Key is a basic method to use in map based comparisons
+// where delimiter or other formatting do not get considered
+func (p *Pair) Key() (PairKey, error) {
```

Comment:
I initially had that, but it blew out somewhat because there is nothing stopping a pair.Base or pair.Quote from being reassigned and then when you call `pair.Key` you may get an inaccurate response

Reply:
Yeah, that is a good point.

Reply:
Upon looking, in order to check if the `PairKey` is correct, you essentially allocate a new pair key to compare against when you may as well just return a `PairKey`.

Rather than this Pairkey concept, do you think stealing your pair.EMPTYFORMAT from your other PR would be a better concept? Doesn't need error processing